SpatialGDK/Source/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -981,7 +981,7 @@ USpatialNetConnection* USpatialNetDriver::AcceptNewPlayer(const FURL& InUrl, boo
 		{
 			// Most of this is taken from "World->SpawnPlayActor", excluding the logic to spawn a pawn which happens during
 			// GameMode->PostLogin(...).
-			APlayerController* NewPlayerController = GameMode->SpawnPlayerController(ROLE_AutonomousProxy, FVector::ZeroVector, FRotator::ZeroRotator);
+			APlayerController* NewPlayerController = GameMode->SpawnPlayerController(ROLE_AutonomousProxy, TEXT(""));
```

Comment:
This is awful.

Reply:
Agreed.

Reply:
Could we put anything meaning full for that string? I have no idea what the parameters are.